### PR TITLE
Increase tolerance of muxing error to account for AV frame mismatch

### PR DESCRIPTION
**Pull request by BrandenAvalonCx**
### Description
Mixing almost always produces an error now as typical recordings often have a slight 2-10 frame mismatch between audio and video. Since this is imperceptible, greatly increase the tolerance as the goal is to flag errors in cases where the mismatch will be significant.

### Changes
Increase tolerance scope

### Validation and Acceptance Criteria
- Manual testing only

### CODE REVIEW

File: common/processors/file_output/FileOutputProcessor.cpp [hunk 1]
```diff
@@ -306,7 +306,10 @@ void FileOutputProcessor::checkAudioVideoDurationMismatch(
   if (!(sampleRate_ > 0)) {
     return;
   }
-  constexpr double kDurationMismatchToleranceSec = 0.05;
+  // Failure tolerance is reasonably large to avoid false positives as 0.5s of
+  // silence is unlikely to be noticed and video/audio recordings seem to often
+  // have a mismatch of ~2-10 frames, which is ~0.05-0.2s at 48kHz.
+  constexpr double kDurationMismatchToleranceSec = 0.5;
```
Comment (ilpoacx) [inline common/processors/file_output/FileOutputProcessor.cpp:312]: [MEDIUM] No test pins the new 0.5s tolerance.

The existing duration-mismatch cases in the `FileOutputProcessor` unit tests exercise gaps of roughly 3.75s and 1.56s. Both sit far outside the old 0.05s tolerance and remain far outside the new 0.5s one, so neither is a boundary case. No test asserts a mismatch magnitude inside the 0.05s-0.5s band, which is exactly the band this change alters.

The practical consequence: a green build here is not evidence that the new threshold behaves as intended, only that the unrelated existing magnitudes still pass. Nothing in the suite would fail if this constant were reverted to 0.05, or raised to 5.0.

CONTRIBUTING.md ("Testing") asks that changes include unit tests validating the change where possible, and the surrounding cases in this file already follow a convention of documenting the magnitude each one exercises relative to the tolerance.

Suggested: add a case that renders an export whose audio/video drift lands between 0.05s and 0.5s and asserts `kNoError`, plus one just above 0.5s asserting the mismatch error still fires. Together those pin both sides of the new boundary, and the first would fail if the constant were reverted.

File: common/processors/file_output/FileOutputProcessor.cpp [hunk 2]
```diff
@@ -306,7 +306,10 @@ void FileOutputProcessor::checkAudioVideoDurationMismatch(
   if (!(sampleRate_ > 0)) {
     return;
   }
-  constexpr double kDurationMismatchToleranceSec = 0.05;
+  // Failure tolerance is reasonably large to avoid false positives as 0.5s of
+  // silence is unlikely to be noticed and video/audio recordings seem to often
+  // have a mismatch of ~2-10 frames, which is ~0.05-0.2s at 48kHz.
```
Comment (ilpoacx) [inline common/processors/file_output/FileOutputProcessor.cpp:311]: [LOW] The frame-to-seconds conversion cites the wrong clock.

"~2-10 frames, which is ~0.05-0.2s at 48kHz" mixes two clocks: frame duration follows the video frame rate, whereas 48kHz is the audio sample rate used for `kAudioDurationSec` two lines below.

Working the arithmetic for common video frame rates, 2-10 frames is:

- 0.083s to 0.417s at 24fps
- 0.080s to 0.400s at 25fps
- 0.067s to 0.333s at 30fps
- 0.033s to 0.167s at 60fps

None of those matches the stated ~0.05-0.2s range: the low end falls below 0.05s and the high end at 24/25fps exceeds 0.2s.

This does not affect the correctness of the value chosen. Ten frames at the lowest common rate (24fps, 0.417s) still fits under 0.5s, so the tolerance is well sized. Only the stated reasoning is off.

Suggested: drop "at 48kHz" and either make the frame-rate assumption explicit ("2-10 frames is roughly 0.07s to 0.42s at 24-30fps") or keep the rationale qualitative.

**Review by ilpoacx — APPROVED** [review]
## Review: widen the duration-mismatch tolerance to 0.5s

**Objective:** raise the audio/video duration-mismatch warning tolerance from 0.05s to 0.5s so that the frame-level drift normally present in recorded video stops raising a false-positive export warning.

**Recommendation: approve.** The change meets that objective, and 0.5s leaves headroom for a 10-frame drift even at 24fps (0.417s). The constant is used symmetrically in both comparison directions, and there is no precision concern at this magnitude. Two non-blocking suggestions are left inline.

| Severity | Count |
|---|---|
| CRITICAL | 0 |
| HIGH | 0 |
| MEDIUM | 1 |
| LOW | 1 |

### Advisory findings

- **[MEDIUM]** line 312 - no test pins the new threshold, so a green build does not demonstrate the new behavior.
- **[LOW]** line 311 - the frame-to-seconds conversion in the new comment cites the audio sample rate rather than the video frame rate.

Details are inline on the relevant lines.
